versions.cfg

```diff
@@ -5,8 +5,9 @@ zc.buildout =
 plone.restapi =
 
 # code analysis
-black = 20.8b1
+black = 22.3.0
```

Comment:
With version 22.3.0 of `black`, no need to pin `click = 8.0.4`. See https://github.com/psf/black/pull/2966

Reply:
Wouldn't it be better to pin the `black` in a constraints.txt? `black` is not used in buildout.

Reply:
Though, on second thought we are about to move away from buildout in the long run, so it might be worth to consider. Will look into it...